src/main/java/com/revature/controllers/SwaggerfileController.java

```diff
@@ -96,4 +110,23 @@
         return ResponseEntity.ok(swaggerSummaryDtos);
     }
     
+    @GetMapping("/csv/{id}")
+    public ResponseEntity<byte[]> getResultSummaryCsv(@PathVariable("id") int id) {
+        ResultSummary rs = resultSummaryService.getById(id);
+        if (rs == null) {
+            return ResponseEntity.notFound().build();
+        }
+        
+        ResultSummaryCsv csv = rs.getResultSummaryCsv();
+        if (csv == null) {
+            return ResponseEntity.notFound().build();
```

Comment:
Preferably, this should return a 204 No Content instead of a 404 Not Found.

Reply:
This has been changed to a 204 No Content